Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -248,6 +264,20 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
   }
 }
 
+void MenuBar::UpdateToolsMenu(bool emulation_started)
```

Comment:
This logic looks different than in DolphinWX, which is (if I'm reading it correctly):

```c++
const auto& sys_menu_loader = DiscIO::NANDContentManager::Access().GetNANDLoader(
    Titles::SYSTEM_MENU, Common::FROM_CONFIGURED_ROOT);
for (auto* action : m_perform_online_update_menu->actions())
  action->setEnabled(!sys_menu_loader.IsValid());
m_perform_online_update_for_current_region->setEnabled(sys_menu_loader.IsValid());
```

Is that intended?